doc/index.rst

```diff
@@ -187,6 +187,7 @@ Note that some of these currently tied only to a single option, but we still lea
 * `USE_DEB` (*DEPRECATED*: use `UPSTREAM_WORKSPACE` instead. default: true): if `true`, `UPSTREAM_WORKSPACE` will be set as `debian`. if `false`, `file` will be set. See `UPSTREAM_WORKSPACE` section for more info.
 * `USE_MOCKUP` (default: not set): reletive path to mockup packages to be used for the tests
 * `VERBOSE_OUTPUT` (default: not set): If `true`, build tool (e.g. Catkin) output prints in verbose mode.
+* `WRITABLE_SOURCE` (default: false): Set this to true if you need the target source to be writable at build (not recommended).
```

Comment:
`target source` could better describe what actually points to. `target source in your workspace`?

Reply:
`the source in  TARGET_REPO_PATH`?
(all other src folders are writebale anyway..)

Reply:
Maybe catkin tools or CMake already has a variable name for this and we can reuse it ? like PROJECT_SOURCE_DIR or so? https://cmake.org/Wiki/CMake_Useful_Variables

Reply:
+1 for `PROJECT_SOURCE_DIR`. That sounds more common.

Meanwhile `TARGET_REPO_PATH` is probably an equivalent in `industrial_ci`. Either way, it may not be something that is exposed to end users is it as an API, except in document like this thread? So we could keep `TARGET_REPO_PATH` in the source code while using `PROJECT_SOURCE_DIR` in document, as an idea.